### ISSUE TITLE
Monitor-specific API Keys are called/organized randomly and inconsistently

### ISSUE THREAD

**Issue by d19dotca** [davidsommer/uptime-dash#1]
Hi there. Love this project and it's a huge improvement over the source, so THANK YOU! :-)

Unfortunately I noticed that when calling monitor-specific API keys, it seems to generate the page with them randomly. They are never in the same order I added them but I would like them to be. Is there anything I can do in the code to fix that?

Your help would be greatly appreciated! :-) Thanks again!

**Comment by davidsommer** [comment]
Hei Dustin
Thx for the Feedback.
Did you mean you want to add the Monitors ID's that should be displayed? If yes, then you would need to add the ID's to the fetch the request:
var url = "http://api.uptimerobot.com/getMonitors?apiKey=" + apikey + "&customUptimeRatio=1-7-30-365&format=json&logs=1&monitors=ID1-ID2-ID3";

If thats not what you meant, please give me an example.

**Comment by d19dotca** [comment]
I will try that, thank you. What I meant was that after setting it up the way it was configured for, after every time I refreshed the web page, the order of the monitors it displayed was changed every time. So ultimately I was hoping to hard-code or sort the monitors by status or alphabetically, etc. I'll give what you suggested a go and see how it works. I kind of stopped using it shortly afterwards because I couldn't sort it. ;-)
